### ISSUE TITLE
Timers: Incorrect freeze time for Tormented Demons

### ISSUE THREAD

**Issue by tylerwgrass** [runelite/runelite#18025]
### Type

Incorrect behavior

### Operating System

None

### Bug description

Tormented demons have a special attack which roots the player for a few frames.
The timer displays the freeze time for entangle.

When casting, the message `The demon's spell binds you!` appears in the chatbox.

### RuneLite version

Runelike version 1.10.34
Launcher version 2.7.1

**Comment by YvesW** [comment]
If you want to, you can check if it works correctly on EHC. If so, it'll likely be fixed in my next buff bar PR.

**Comment by tylerwgrass** [comment]
Seems okay in EHC, I can check again after your PR is merged. Mind linking it here?
I'll see if I can gather any other info on them later as well.

**Comment by YvesW** [comment]
Apologies, I should have been more clear on Discord. I haven't checked out the bound vars yet. I do not know if it's transmitted, and if so, how it behaves. I might be able to take a look in a couple of hours. Feel free to @ me on Discord for further follow-up.

**Comment by Nightfirecat** [comment]
I don't think we can fix this easily as they use a shared animation (entangle), and the client doesn't have information about the length of a player's freeze timer.

**Comment by tylerwgrass** [comment]
Could the chat message be used? Seems [something similar](https://github.com/runelite/runelite/blob/cb48373da0c5a2042bd32938c6a1086a8e29c832/runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java#L850) is done with ice barrage.
1) Determine player is bound from the chat message.
2) Use a newly defined timer [here](https://github.com/runelite/runelite/blob/cb48373da0c5a2042bd32938c6a1086a8e29c832/runelite-client/src/main/java/net/runelite/client/plugins/timers/GameTimer.java#L41) after getting the number of ticks the player is bound manually.
3) Ignore creating the entangle timer if this one is active (better way to do this?)

**Comment by tylerwgrass** [comment]
Partially resolved with [1b0b2d9](https://github.com/runelite/runelite/commit/1b0b2d9b3538ea8f2270bd714aa094a0660471af) but maybe good enough to close this?

The timer still stays if you don't move and tank the special attack, but doubt people are going to be doing that
